src/cs/MonoGame.Extended.Collisions/CollisionComponent.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.Xna.Framework;
 using MonoGame.Extended.Collisions.Layers;
 using MonoGame.Extended.Collisions.QuadTree;
+using Newtonsoft.Json.Linq;
```

Comment:
Is MonoGame.Extended trying to stay AOT compliant?

This article on MonoGame says that Newtonsoft json are not AOT compliant for console?  Also that Linq should be avoided?  Also, why Newtonsoft's Linq vs System.Linq?

https://docs.monogame.net/articles/preparing_for_consoles.html